src/Temporalio/Bridge/Worker.cs

```diff
@@ -16,11 +17,16 @@ internal class Worker : SafeHandle
         /// <param name="client">Client for the worker.</param>
         /// <param name="namespace_">Namespace for the worker.</param>
         /// <param name="options">Options for the worker.</param>
+        /// <param name="loggerFactory">Logger factory, used instead of the one in options by
+        ///   anything in the bridge that needs it, since it's guaranteed to be set.</param>
         /// <exception cref="Exception">
         /// If any of the options are invalid including improperly defined workflows/activities.
         /// </exception>
         public Worker(
-            Client client, string namespace_, Temporalio.Worker.TemporalWorkerOptions options)
+            Client client,
+            string namespace_,
+            Temporalio.Worker.TemporalWorkerOptions options,
+            ILoggerFactory loggerFactory)
```

Comment:
This logger factory is already on the client and technically overridable on the worker. So you can probably remove this param and add something like `var loggerFactory = options.LoggerFactory ?? client.Options.LoggerFactory`.

Reply:
Options aren't accessible on that Client type

Reply:
My mistake, got my clients mixed up